src/appleseed.shaders/src/alshaders/alSurface.osl

```diff
@@ -389,39 +388,45 @@ shader alSurface
 
         float sssIor = 1.3;
 
-        if (in_sssMode == SSSMODE_DIFFUSION || in_sssMode == SSSMODE_DIRECTIONAL)
+        if (in_sssMode == BSSRDF_MODE_DIFFUSION ||
+            in_sssMode == BSSRDF_MODE_DIRECTIONAL)
         {
             // TODO: implement this.
         }
-        else // SSSMODE_CUBIC || SSSMODE_EMPIRICAL
+        else // BSSRDF_MODE_CUBIC || BSSRDF_MODE_EMPIRICAL
         {
-            if (sssWeight1 > IMPORTANCE_EPS)
+            if (sssWeight1 > IMPORTANCE_THRESHOLD)
             {
                 color radius = max(
                     color(0.0001),
-                    in_sssRadius * in_sssRadiusColor / (in_sssDensityScale * 7.0));
-
-                out_result += sssColor * sssWeight1 * as_subsurface(
-                    in_sssMode == SSSMODE_CUBIC ? "gaussian" : "normalized_diffusion",
+                    in_sssRadius * in_sssRadiusColor /
```

Comment:
I suggest writing this expression as a single line, so that the two arguments of `max()` become easier to parse:

    max(
        color(0.0001),
        in_sssRadius * in_sssRadiusColor / (in_sssDensityScale * 7.0))